integration-cli/docker_cli_run_test.go

```diff
@@ -3468,3 +3468,27 @@ func TestRunContainerWithRmFlagCannotStartContainer(t *testing.T) {
 
 	logDone("run - container is removed if run with --rm and cannot start")
 }
+
+func TestRunPidHostWithChildIsKillable(t *testing.T) {
+	defer deleteAllContainers()
+	name := "ibuildthecloud"
+	if out, err := exec.Command(dockerBinary, "run", "-d", "--name", name, "busybox", "sh", "-c", "sleep 300; echo hi").CombinedOutput(); err != nil {
```

Comment:
I think it was just an exact duplication of https://github.com/docker/docker/issues/11087